gradle/maven-to-gradle/pom.xml

```diff
@@ -0,0 +1,15 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This file is using tabs, please convert to spaces.

Reply:
I tried to fix it. Please let me know if it is fine now.

Reply:
Looks good!